discovery_engine_core/core/src/engine.rs

```diff
@@ -81,7 +81,7 @@ pub struct InitConfig {
     pub api_key: String,
     /// API base url.
     pub api_base_url: String,
-    /// List of markets to use.
+    /// List of markets to use
```

Comment:
is this reverted on purpose? all other field docs had a period added recently.

Reply:
perhaps a mix up while fixing merge conflicts - thanks.